crates/rethnet_rpc_server/build.rs

```diff
@@ -0,0 +1,35 @@
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+#[derive(Debug, Serialize, Deserialize)]
+struct CargoToml {
```

Comment:
A crate exists for this: https://crates.io/crates/cargo_toml

Reply:
good call, thanks, i used it in https://github.com/NomicFoundation/hardhat/commit/257f7c8c1f10cb969a2db5355ecf5aef6c72261e